test/SILGen/magic_identifier_file.swift

```diff
@@ -12,6 +12,15 @@
 // Test in "Swift 6" mode:
 // RUN: %target-swift-emit-silgen -I %t -enable-experimental-concise-pound-file -module-name Foo %/s | %FileCheck --check-prefixes=BOTH,CONCISE %s
 
+// Remove compiled modules so we test against interfaces:
+// RUN: rm %t/MagicIdentifierFileSwift5.swiftmodule %t/MagicIdentifierFileSwift6.swiftmodule
+
+// Test in Swift 5 mode:
+// RUN: %target-swift-emit-silgen -I %t -module-name Foo %/s | %FileCheck --check-prefixes=BOTH,ABSOLUTE %s
```

Comment:
Is there a particular reason to use `%/s` for the first command?

Reply:
If I used %s, the path would contain backslashes on Windows, and those backslashes would end up in the strings containing `SOURCE_DIR`. I would therefore have to replace the `/`es on those lines with {{/|\\\\}} or whatever.